### PR TITLE
Feature/psc not ceased

### PR DESCRIPTION
**Pull request by clewis1**
https://companieshouse.atlassian.net/browse/PSC-112

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/pscfiling/api/validator/PscIsActiveValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            new FilingValidationContext(dto, errors, transaction, pscType, passthroughHeader));
+
+        assertThat(errors, is(empty()));
+    }
+
+    @Test
+    void validateWhenPSCisNotActive() {
+
+        var fieldError = new FieldError("object", "ceased_on", CEASED_ON, false,
+            new String[]{null, "notMatch.ceased_on"}, null,
```
Comment (PhilipGlover) [inline src/test/java/uk/gov/companieshouse/pscfiling/api/validator/PscIsActiveValidatorTest.java:72]: `"notMatch.ceased_on"` - I'm not sure what this should be, but I suspect it should be more like `"date.ceased_on"`

File: src/main/java/uk/gov/companieshouse/pscfiling/api/validator/PscIsActiveValidator.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        final PscApi pscDetails;
+        pscDetails = pscDetailsService.getPscDetails(validationContext.getTransaction(),
+            validationContext.getDto().getReferencePscId(), validationContext.getPscType(),
+            validationContext.getPassthroughHeader());
+
+        if (Optional.ofNullable(pscDetails.getCeasedOn()).isPresent()) {
+            validationContext.getErrors()
+                .add(new FieldError("object", "ceased_on", validationContext.getDto().getCeasedOn(),
+                    false, new String[]{null, "notMatch.ceased_on"}, null,
```
Comment (PhilipGlover) [inline src/main/java/uk/gov/companieshouse/pscfiling/api/validator/PscIsActiveValidator.java:29]: `"notMatch.ceased_on"` this seems not quite right to me.  `"date.ceased_on"` ???

Comment (clewis1) [reply src/main/java/uk/gov/companieshouse/pscfiling/api/validator/PscIsActiveValidator.java:29]: Agreed. Changes made.

**Review by PhilipGlover — APPROVED** [review]
- tests pass
- have run and error returned:
```
{
    "errors": [
        {
            "error": "PSC is not active as a ceased on date is present",
            "error_values": {
                "rejected": "2022-03-05"
            },
            "location": "$.ceased_on",
            "location_type": "json-path",
            "type": "ch:validation"
        }
    ]
}
```

**Pull request merged**